src/Microsoft.SqlTools.ServiceLayer/Profiler/ObservableXEventSession.cs

```diff
@@ -143,6 +143,17 @@ private Task OnEventRead(IXEvent xEvent)
             {
                 profileEvent.Values.Add(kvp.Key, kvp.Value.ToString());
             }
+            // Add the XE 'actions'.
+            foreach (var kvp in xEvent.Actions)

```

Comment:
I assume `Actions` can never be null here?

Reply:
I will add a explicit check for this, otherwise this would generate exception